src/index.ts

```diff
@@ -317,6 +317,19 @@ export default class CodeTool implements BlockTool {
         case 'Tab':
           this.tabHandler(event);
           break;
+        case 'Backspace':
+          if (textarea.value.length > 0) {
+            event.stopPropagation();
+          } else {
+            event.preventDefault();
+          }
+          break;
+        case 'ArrowUp':
+        case 'ArrowDown':
+        case 'ArrowLeft':
+        case 'ArrowRight':
+          event.stopPropagation();
```

Comment:
why do need this?

Reply:
For navigating through textarea, we use cursor keys and the reason for using `stopPropagation()` is same as above.